Makefile

```diff
@@ -66,4 +67,22 @@ sdk/update:
 
 	@if [ $(commit) = true ]; then \
 		./scripts/open-pr.sh $(service); \
+	fi
+
+# This recipe pulls the specs for the given service from locally cloned cloud-api and re-generates the service's go clients.
+.PHONY: sdk/update-local # service=cloud-foo-service
+sdk/update-local:
+	@if [ -z $(service) ]; then \
+		echo "ERROR: No service argument provided, please provide in the format 'service=...'" >&2; \
+  		exit 1; \
+	fi
+
+	bash ./scripts/pull-specs-local.sh $(service);
+
+	@if [ $(service) = "cloud-shared" ]; then \
+		echo "Generating latest SDK for cloud-shared"; \
+		bash ./scripts/gen-go-shared-sdk.sh $(service); \
```

Comment:
Do we need to pass in the service for this one? Couldn't tell from a quick look at the script.

Reply:
nope, good catch